scrapers/il/bills.py

```diff
@@ -279,19 +279,22 @@ def chamber_slug(chamber):
 
 
 class IlBillScraper(Scraper):
-    LEGISLATION_URL = "https://ilga.gov/legislation/grplist.asp"
+    LEGISLATION_URL = "https://beta.ilga.gov/Legislation/"
     localize = pytz.timezone("America/Chicago").localize
 
     def get_bill_urls(self, chamber, session, doc_type):
         params = session_details[session]["params"]
-        params["num1"] = "1"
-        params["num2"] = "10000"
-        params["DocTypeID"] = doc_type
-        html = self.get(self.LEGISLATION_URL, params=params).text
+        url = "https://beta.ilga.gov/Legislation/RegularSession/{}?SessionId={}".format(
```

Comment:
Please use a `BASE_URL` constant (or similar variable) in place of the literal string `https://beta.ilga.gov`

Reply:
Fixed issue